D4Companion.Services/ScreenProcessHandler.cs

```diff
@@ -683,7 +683,15 @@ private bool FindItemAffixes()
 
             string affixPreset = _settingsManager.Settings.SelectedAffixName;
             var itemAffixes = _affixPresetManager.AffixPresets.FirstOrDefault(s => s.Name == affixPreset)?.ItemAffixes;
+
+            var UseAny = _settingsManager.Settings.UseAny;
+            if (UseAny)
+            {
+                _currentTooltip.ItemType = "any_rare";
+            }
+            
             var itemAffixesPerType = itemAffixes?.FindAll(itemAffix => _currentTooltip.ItemType.StartsWith($"{itemAffix.Type}_"));
```

Comment:
Only line 693 is needed. Instead of using the StartsWith filter you can just return all affixes in `itemAffixes`. No need to set _currentTooltip.ItemType to "any_rare";